src/resources/js/components/layout/UserTooltip.vue

```diff
@@ -1,11 +1,100 @@
 <template>
-    <v-btn icon>
-        <v-icon>mdi-account-circle</v-icon>
+<v-container>
+    <v-btn
+    icon
+    v-if="isLoaded"
+    @click="beheavior">
+        <v-icon>{{icon}}</v-icon>
     </v-btn>
+    </v-container>
 </template>
 
 <script>
+let alert = {};
 export default {
+  props: ['parentRefs'],
   name: 'UserTooltip',
+
+  data() {
+    return {
+      icon: '',
+      isLoaded: false,
+      isLogged: false,
+    };
+  },
+  beforeMount() {
+    alert = this.parentRefs.alert; // without this line, there are "uncaught exception: Object"
+    this.refresh()
+      .then(() => {
```

Comment:
watch out your possibly calling a `.then` after a `.catch` following the axios workflow the unique method chainable after a `.catch` should be a `.else`